src/components/SearchForm/SearchForm.js

```diff
@@ -35,16 +35,16 @@ function SearchForm({ isLoggedIn }) {
   };
 
   return (
-    <div className="Search-Form">
-      <div className="Search-Form__search-bar">
+    <section className="search-form"> 
+      <div className="search-form__search-bar">
         <input
           type="text"
-          className="Search-Form__search-input"
+          className="search-form__search-input"
           placeholder="Enter topic"
           onChange={handleInputChange}
           onKeyDown={handleKeyPress}
```

Comment:
Please, delete `handleKeyPress` from `SearchForm` 

Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses